.coveragerc

```diff
@@ -1400,6 +1400,9 @@ omit =
     homeassistant/components/zwave/util.py
     homeassistant/components/zwave_js/discovery.py
     homeassistant/components/zwave_js/sensor.py
+    homeassistant/components/zwave_me/__init__.py
+    homeassistant/components/zwave_me/helpers.py
+    homeassistant/components/zwave_me/number.py
```

Comment:
Please make sure that all the modules that aren't covered by tests are excluded from coverage calculation. We only have tests for the config flow.

Reply:
https://github.com/home-assistant/core/pull/65491

Reply:
https://github.com/home-assistant/core/pull/65491